File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    # 길이 확인하기
    length = len(s)
    if length != 4 and length != 6:
        return False
    # 숫자인지 확인하기
    for c in s:
        if not c.isnumeric():
            return False
    return True

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        /*
        길이 check
        숫자 check
        */
        
        if(!(s.length() == 4 || s.length() == 6)){
            return false;
        }
        for(char ch: s.toCharArray()){
            if(ch < '0' || ch > '9'){
                return false;
            }
        }
        return true;
    }
}